glslc/README.asciidoc

```diff
@@ -208,12 +208,9 @@ empty value.
 
 ==== `-I`
 
-`-I` adds the specified directory to the search path for include files.
-
-Only `""` style includes are permitted. When searching for included files, first
-the directory of the current file is attempted. If this fails each directory
-supplied via `-I` on the command line is searched in the given order for the
-file.
+`-Idirectory` adds the specified directory to the search path for include files.
+The directory may be an absolute path or a relative path to the location where
+`glslc` is run.  It must use path separators that the OS supports.
```

Comment:
the bit about "relative path to the location where glslc is run"  sounds like it could be mis-read as the path to glslc.  Could we be more clear by saying "the current working directory"?

Secondly, "It must use path separators that the OS supports" seems... redundant?